### PR TITLE
Replace the existing datetime microformat with the HTML5 time element

### PR DESCRIPTION
**Pull request by robinwhittleton**
The datetime microformat has been superceded by the [HTML5 time element](http://www.w3.org/TR/html51/semantics.html#the-time-element). They offer better automated parsing (time elements can be easily extracted), better semantics (no misuse of the abbr element) and better accessibility (users with title reading turned on in their screenreader hear effectively a random string of numbers and letters with datetime microformats).

### CODE REVIEW

**Comment by fofr** [comment]
Oooh, awesome. This has been on my todo list for a while. :heart: 

Changes look good to me :+1:. @edds might want to have a look too.

**Comment by robinwhittleton** [comment]
I’ve also snuck in one more fix for an issue I noticed in Firefox while testing this. I’ve applied the same fix to static in a [separate PR](https://github.com/alphagov/static/pull/639).

**Comment by tombye** [comment]
I can't comment on the changes but the intention and description are great, awesome work.

File: app/assets/stylesheets/frontend/reset/_reset.scss [hunk 1]
```diff
@@ -30,3 +30,10 @@ table, caption, tbody, tfoot, thead, tr, th, td {
   font-weight: normal;
 }
 
+strong {
+  font-weight: bold;
+}
```
Comment (edds) [inline app/assets/stylesheets/frontend/reset/_reset.scss]: What is the rational for adding this? We purposefully haven't added this in the past.

Comment (robinwhittleton) [inline app/assets/stylesheets/frontend/reset/_reset.scss]: Should have linked that commit to [static’s reset file](https://github.com/alphagov/static/blob/master/app/assets/stylesheets/helpers/_reset.scss), where you [purposefully added this in the past](https://github.com/alphagov/static/commit/2184c82e45c186c71f12b026209eca610f816228).

Comment (edds) [inline app/assets/stylesheets/frontend/reset/_reset.scss]: We specifically don't want to give govspeak parts of whitehall the ability to have bold. So can we either add an override to this in the govspeak or not add this here.

Comment (robinwhittleton) [inline app/assets/stylesheets/frontend/reset/_reset.scss]: No problem, removed from the patch.

**Comment by edds** [comment]
:+1:

**Pull request merged**